chemprop/cli/predict.py

```diff
@@ -282,12 +277,9 @@ def make_prediction_for_models(
         individual_preds.append(preds)
 
     average_preds = torch.mean(torch.stack(individual_preds).float(), dim=0)
-    if args.target_columns is not None:
-        assert (
-            len(args.target_columns) == model.n_tasks
-        ), "Number of target columns must match the number of tasks."
-        target_columns = args.target_columns
-    else:
+
+    output_columns = load_output_columns(model_paths[0])
+    if output_columns is None:
         target_columns = [
```

Comment:
```suggestion
        output_columns = [
```

Reply:
There are some `target_columns` that have not been replaced with `output_columns` in the file.

Reply:
Shall we also change the name in other modules?

Reply:
I can see if both way. For training, these are _targets_ for the model to learn, while for predicting, these are model _outputs_. So I think the current distinguished names make sense.